geopmdpy/test/TestAccessLists.py

```diff
@@ -203,23 +199,23 @@ def test_get_group_access_default(self):
         self.assertEqual(set(self._controls_expect), set(controls))
 
     def test_get_group_access_named(self):
```

Comment:
Minor note: maybe this test name should change. Same note applies to `test_set_group_access_named`.

Also, perhaps there should be another test that validates the new flexible behavior in which `geopmaccess` accepts either a group name or ID.

Reply:
We don't currently get coverage of geopmdpy.access.main where the CLI is parsed, and the group is converted.  That is a bigger problem...I've updated the names

Reply:
Moved the implementation into function outside of main() called geopmdpy.access._group_id() and wrote a unit test that calls it.